packages/spectacle/src/components/code-pane.tsx

```diff
@@ -12,7 +12,11 @@ import { useSteps } from '../hooks/use-steps';
 import indentNormalizer from '../utils/indent-normalizer';
 import styled, { ThemeContext } from 'styled-components';
 import { compose, layout, position } from 'styled-system';
-import dark from 'react-syntax-highlighter/dist/cjs/styles/prism/vs-dark';
+/**
+ * The types for these theme files don't exist, so we need to ts-ignore it.
+ */
+// @ts-ignore
+import defaultTheme from 'react-syntax-highlighter/dist/esm/styles/prism/vs-dark.js';
```

Comment:
This breaks Node in CJS mode. Just the following won't work:

```
$ node 
Welcome to Node.js v16.16.0.
Type ".help" for more information.
> const Spectacle = require("./packages/spectacle/lib/index.js")
/Users/rye/scm/fmd/spectacle/node_modules/.pnpm/react-syntax-highlighter@15.5.0_react@18.2.0/node_modules/react-syntax-highlighter/dist/esm/styles/prism/vs-dark.js:3
export default {
^^^^^^

Uncaught SyntaxError: Unexpected token 'export'
```

Reply:
However, as a reminder ESM can import CJS just fine, so maybe go for the CJS version like we had before?

Reply:
Shoot. Yeah it needs the extension to work with modules. This one has always been a pain since it's not a real export, but we're just grabbing it from the packages' path.

```
Module not found: Error: Can't resolve 'react-syntax-highlighter/dist/esm/styles/prism/vs-dark' in '/Users/carloskelly13/Developer/spectacle/packages/spectacle/lib'
Did you mean 'vs-dark.js'?
```

without the extension. Maybe we just include this file or make our own theme.

Reply:
Also, now that I'm thinking about it we should add a CI test that with appropriate deps+scripts for wireit effectively does:

```sh
$ cd packages/spectacle
$ pnpm build:lib
$ node -e "require('./lib/index.js')"
```

(This presently fails)

Reply:
I think the solution here it to vendor this theme ourselves since neither import works.

Reply:
I would think that adding extension to CJS would work?

```
import dark from 'react-syntax-highlighter/dist/cjs/styles/prism/vs-dark.js';
```

Reply:
You're right it does. I was missing the `.default` on the usage since it's not a module. `node -e "require('./lib/index.js')"` runs clean now. Thank you!